libs/core/src/lib/facets/facet/facet.component.html

```diff
@@ -0,0 +1,25 @@
+<ng-container *ngIf="type === 'image'">
+    <ng-container *ngTemplateOutlet="customContentTemplate"></ng-container>
+</ng-container>
+
+<ng-container *ngIf="type !== 'image'">
+    <h1 *ngIf="title" fd-title headerSize="5" class="fd-margin-bottom--sm" [id]="titleId">{{ title }}</h1>
+    <ng-container *ngIf="type === 'form'">
+        <ng-container *ngTemplateOutlet="contentTemplate"></ng-container>
+    </ng-container>
+    <ng-container *ngIf="type === 'key-value' || 'custom'">
+        <ng-container *ngTemplateOutlet="customContentTemplate"></ng-container>
+    </ng-container>
+    <ng-container *ngIf="type === 'rating-indicator'">
+        <h2 class="fd-form-label">{{ subtitle }}</h2>
```

Comment:
Could you add `fd-form-label` direcive there, instead of class?

Reply:
replaced with `fd-form-label` directive.